build.sbt

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Spurious newline.

Reply:
Ah yeah, residue of merge, will resolve, thanks.